### ISSUE TITLE
Windows 7 touch doesn't work with WinForms

### ISSUE THREAD

**Issue by MihaMarkic** [cefsharp/CefSharp#1296]
Anybody else noticed that touch in Windows 7 doesn't work even with WinForms version (doesn't scroll, acts more like a mouse)? Whereas Windows 10 works fine (I suppose 8.x as well). Chromium standalone works fine. 
Now, I didn't spend much time on this.

**Comment by amaitland** [comment]
Please include a version when creating an issue.

**Comment by MihaMarkic** [comment]
Sorry, it is 43.0.0-pre02. I just observed this behavior on my project where I host WinForms cef in a WPF app. I didn't manage yet to test the WinForms sample.

**Comment by amaitland** [comment]
> I just observed this behavior on my project where I host WinForms cef in a WPF app.

Anything else we should be aware of? 

> Chromium standalone works fine.

Did you compare an equivalent version of `Chrome`?

**Comment by MihaMarkic** [comment]
> Anything else we should be aware of? 
Can't think of anything else now.
> Did you compare an equivalent version of Chrome?
Unfortunately not, just with the latest. My bad. Will try it.

**Comment by amaitland** [comment]
> Unfortunately not, just with the latest. My bad. Will try it.

Testing with `cefclient` of the same version should be enough.

https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-cef

There's also the new `2454` version (the `master` branch currently under development).

**Comment by MihaMarkic** [comment]
OK, I did some more tests. CEF2357 works fine and so does pure WinForms sample. That leaves me to a conclusion that it is something related to hosting a WinForms control within WPF app on Windows 7. The funny thing is that I am hosting WinForms in WPF just because of the touch support.
Well never mind, it isn't that important as the target machine will probably switch to Windows 10.
Hopefully touch support for WPF is implemented in the future.

**Comment by amaitland** [comment]
> Hopefully touch support for WPF is implemented in the future.

Still waiting on `upstream` support to be merged. See https://bitbucket.org/chromiumembedded/cef/issues/1059

**Comment by amaitland** [comment]
Duplicate of #1833
